app/assets/stylesheets/boards.scss

```diff
@@ -0,0 +1,34 @@
+.board {
+ 	width: 95%;
+  height: 95%;
+  border: 3px solid #333;
+  margin: 10px auto 10px auto;
+  vertical-align: middle;
+}
+body {
+ 	padding-top: 10%; 
+ 	padding-bottom: 10%;
+}
+td {
+  empty-cells: show;
+}
+table.one, .board {
+  border-collapse: collapse;
+ 	height: 45em;
+ 	width: 45em;
```

Comment:
Any HTML element can have a border, so you can just put one directly on the table.